src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs

```diff
@@ -63,8 +65,10 @@ public ServiceManagerBuilder WithCallingAssembly()
         /// <returns>The instance of the <see cref="IServiceManager"/>.</returns>
         public IServiceManager Build()
         {
-            _services.AddSignalRServiceManager();
-            return new ServiceManager(_services);
+            return _services.AddSignalRServiceManager()
+                .AddSingleton(_services.ToList() as IReadOnlyCollection<ServiceDescriptor>)
```

Comment:
since ServiceManager is useless for multi-endpoint scenario, how about exclude ServiceManager, and use a new builder, then the process to build the context should be simple
 e.g. 
``` c#
var context = MultiEndpointHubContextBuilder.WithConfiguration(params).Build();
// start
await context.StartAsync();
// negotiate
var res = await context.NegotiateAsync(params);
// send message
await context.Clients.X.SendAsync(params);
// stop
await context.StopAsync();
```

Reply:
@vicancy @vwxyzh what do you think?
I think the service manger is totally useless in muti-endpoint scenario. so directly build a hub context is better